test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -5,6 +5,147 @@
  */
 #include "serializer.h"
 
+#include <stack>
+#include <string>
+
+namespace {
+
+const std::string kCounterNamePrefix = "counter_";
+
+/**
+ * Class that controls id creation for counters. Basically, a variable wrapper
+ * that guarantees variable to be incremented.
+ */
+class CounterIdProvider_ {
```

Comment:
Why `CounterIdProvider_` instead of just `CounterIdProvider`?

Reply:
Initially, my intention was to name the class like a private class field, considering its placement within an anonymous namespace. However, since that concept wasn't clear, I have chosen to remove the trailing underscore. There is actually no convention about naming classes in an anonymous namespaces this way in any company that I am aware of.